src/platform/polyfills/download-attribute.js

```diff
@@ -6,7 +6,7 @@ if (isInternetExplorer) {
   document.addEventListener('click', event => {
     const { target } = event;
 
-    if (target?.tagName === 'A' && target?.getAttribute('download')) {
+    if (target?.tagName === 'A' && target?.hasAttribute('download')) {
```

Comment:
Semi-related change I discovered while testing this work - `download` is always just a property with an empty value (falsy), so we should be using `hasAttribute` instead of `getAttribute`